flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobVertexDetailsInfo.java

```diff
@@ -41,6 +41,7 @@
     public static final String FIELD_NAME_VERTEX_ID = "id";
     public static final String FIELD_NAME_VERTEX_NAME = "name";
     public static final String FIELD_NAME_PARALLELISM = "parallelism";
+    public static final String FIELD_NAME_MAX_PARALLELISM = "max-parallelism";
```

Comment:
new fields should follow `camelCase` (dashes are a legacy format that we're trying to phase out)

Reply:
Done